wikidataQuery.py

```diff
@@ -3,13 +3,17 @@
 import settings
 import traceback
 import json
+import simplejson
+# Installable with 'pip install textblob'
+from textblob import TextBlob
```

Comment:
Apparently I also needed to run `python3 -m textblob.download_corpora` please include that in the README as well.

Reply:
Hm, weird. I didn't need 'python3 -m textblob.download_corpora'